ai/antrea_data_task/Issue Data Processing Document.md

```diff
@@ -0,0 +1,104 @@
+# Antrea Issue Data Processing Document
+
+## Introduction
+
+This document introduces how to process the Issue data from the Antrea project into a standardized format suitable for
+training. Before processing the data, it is necessary to install the corresponding Python libraries. Please run the
+following command in the Conda environment.
+
+```plain
+pip install -r requirements.txt
+```
+
+## How to use ？
+
```

Comment:
Could you add the introduction section to explain the different role of the training datasets and why we need them?

Reply:
Okay, I will handle it.